Sources/SwiftCrossUI/Views/Modifiers/PresentationModifiers.swift

```diff
@@ -0,0 +1,62 @@
+extension View {
+    /// Sets the available detents (heights) for a sheet presentation.
+    ///
+    /// This modifier only affects the sheet presentation itself.
+    /// It allows users to resize the sheet to different predefined heights.
+    ///
+    /// - Supported platforms: iOS 15+ (ignored on unsupported platforms)
+    /// - `.fraction` and `.height` fall back to `.medium` on iOS 15 and earlier
+    ///
+    /// - Parameter detents: A set of detents that the sheet can be resized to.
+    /// - Returns: A view with the presentation detents preference set.
```

Comment:
update this `Returns` and the one below to refer to the preference by property name as you did for the bottom 3 modifiers.